proto/binlogdata.proto

```diff
@@ -130,6 +130,7 @@ message Rule {
 // wins.
 message Filter {
   repeated Rule rules = 1;
+  bool bestEffortNameInFieldEvent = 2;
```

Comment:
This name is kind of clunky, not that I have a better suggestion :(. Maybe think about a shorter name that still conveys the correct meaning.

Reply:
i wanted a name whose default value can be false by default (for backward compatibility). Any suggestion?

Reply:
how about bestEffortFieldEvent?

Reply:
Or an enum called FieldEventMode with values like ERR_ON_MISMATCH and BEST_EFFORT.